src/Options.php

```diff
@@ -701,6 +701,28 @@ public function setMaxRequestBodySize(string $maxRequestBodySize): void
         $this->options = $this->resolver->resolve($options);
     }
 
+    /**
+     * Gets the type serializers.
+     *
+     * @return array
+     */
+    public function getClassSerializers(): array
+    {
+        return $this->options['class_serializers'];
+    }
+
+    /**
+     * Sets the class serializers.
```

Comment:
Maybe a better description would be `Sets a list of callables that will be called to customize how objects are serialized in the event's payload. The list must be a map of FQCN/callable pairs.`